Gemfile.lock

```diff
@@ -1,60 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.5.0)
+    better_errors (2.5.1)
```

Comment:
### [better_errors](https://github.com/BetterErrors/better_errors)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.5.0 → 2.5.1

[_[change-log](https://github.com/BetterErrors/better_errors/releases), [source-code](https://github.com/BetterErrors/better_errors)_]





<details>
<summary>Commits</summary>

A change of **18** commits. See the full changes on [the compare page](https://github.com/BetterErrors/better_errors/compare/v2.5.0...v2.5.1).

These are the first 10 commits:
- (8077c90) [Add rubygems metadata](https://github.com/BetterErrors/better_errors/commit/8077c90dda36e1b510558657092ebd6087dcd914)
- (51bd42a) [Warn on missing metadata support.](https://github.com/BetterErrors/better_errors/commit/51bd42a9df48e073d3cf9932969627ad4bc6898b)
- (ad519b7) [Merge pull request #427 from viraptor/patch-1](https://github.com/BetterErrors/better_errors/commit/ad519b7fb4b7179d38085782521584c2b3b2eaae)
- (893e72e) [Fall back to older rubygems on Ruby 2.2](https://github.com/BetterErrors/better_errors/commit/893e72e365353bbfae2c44817c05a25beade8466)
- (7d7d17a) [Use even older rubygems for Rails 4.2](https://github.com/BetterErrors/better_errors/commit/7d7d17a32fa61ef81ac6e992746b0376d16f3c1b)
- (a7e10cf) [Fix bundler version when using older rubygems](https://github.com/BetterErrors/better_errors/commit/a7e10cf58b2f847896d5aab81d24d471a0741aa6)
- (e4cfa2f) [Another attempt to fix bundler version in CI](https://github.com/BetterErrors/better_errors/commit/e4cfa2f2f0c896e75a19c6ed244c00f8dd144f09)
- (43cea4d) [Fix gem update asking for confirmation](https://github.com/BetterErrors/better_errors/commit/43cea4d3b8d9cf898d3169584be4556928fd871c)
- (01279b9) [Quit mucking with rubygems and bundler version](https://github.com/BetterErrors/better_errors/commit/01279b9d9071b76e6500ff19b349ca15f7a07120)
- (5612121) [Set rubygems and bundler version for rails 4.2](https://github.com/BetterErrors/better_errors/commit/56121218f347059183400dd0f6a6c353a1ea03e4)


</details>